pglookout/logutil.py

```diff
@@ -5,39 +5,50 @@
 Copyright (c) 2015 Ohmu Ltd
 See LICENSE for details
 """
+from __future__ import annotations
```

Comment:
I've made a detailed comment there (private repo):
https://github.com/aiven/aiven-core/pull/31814#discussion_r1140516577

There is some misunderstanding about "the thing" that was supposed to enter the language in 3.10, then 3.11, then indefinitely on hold. "The thing" is not this statement. "The thing" is to make the behaviour activated by this statement, the behaviour by default, without the need to write this statement (rendering this statement a no-op).
However, this statement is there to stay in future versions of Python, and it won't go away. It recommended and pretty future-proof for the time being. More details in the provided link. ☝️

Reply:
> "The thing" is to make the behaviour activated by this statement, the behaviour by default, without the need to write this statement

Yes, that's the typical behaviour of ```__future__``` imports (usually - maybe always? - they're parser directives that allow progressive switching of a codebase to a certain feature), that was clear.

The problem is slightly different; I don't remember any feature being dropped; it's ```__future__```, not ```__experimental___``` or something like that. Usually something lands in ```__future__``` in version and then becomes default behaviour in version X+N with N>=1.

In this case, we don't know what's going to happen. There're chances that the feature will be dropped; it will probably stay in ```__future__``` for a while, but do we want to take that unnecessary risk? We're not scrambling to remove annotations import where we already use it, but we're trying not to add it where we don't use it already.